### DISCUSSION TITLE
3004.388.6_2-gnuton1

### DISCUSSION THREAD

**Discussion by gnuton** [gnuton/asuswrt-merlin.ng#569] [Announcements]
##  GNUton's Asus Merlin changelog  ##

Date: April 04, 2024
Release: 3004.388.6_2-gnuton1

  - ADDED: Support for TUF-AX3000 V2
  - ADDED: Support for RT-AX58U V2

<hr /><em>This discussion was created from the release <a href='https://github.com/gnuton/asuswrt-merlin.ng/releases/tag/3004.388.6_2-gnuton1'>3004.388.6_2-gnuton1</a>.</em>

**Comment by alfky12** [comment]
For tuf-ax3000 (v1) firmware 3004.388.6_2-gnuton1_tuf, I see:
1. Header misalign? (merlin icon with tuf-ax3000 text, logout button)
2. No icon for wifi (located below the language selection button) if it set only partial (2.4 ghz or 5ghz only)
3. No icon for tools in side panel
4. If QoS set to Cake, Classification tab should not show traffic data, but instead it does.

**Reply by gnuton** [reply]
@alfky12 can open the router page from a browser in incognito mode? do you still see these issues?

**Reply by alfky12** [reply]
Yes, I tried it before writing that. I tried with edge, brave, chrome and firefox browsers, in regular and incognito mode, all produce the same display. There are no extensions/addons active in incognito mode.

**Comment by CPngN** [comment]
Is there really no difference between 388.6_2 and 388.4 aside from 2 new models?  Is it worth the upgrade on a pair of ET8's that are on 388.4? (More stable? Updated packages?) Thx

**Comment by tjbbjt** [comment]
Working great so far on a RT-AX58U V2. Thanks so much @gnuton!

**Comment by svindalb** [comment]
Flashed the firmware on my AX82U. After it was done, the router is still on the stock firmware, no errors.

**Reply by gnuton** [reply]
This has been fixed in the alpha

**Comment by lew601** [comment]
AX82U V2 nothing change after it was completed...

**Reply by gnuton** [reply]
There is a fix for this in the alpha

**Comment by SOBON2O** [comment]
I have an Asus RT-AX82U, and I updated to the this {latest} firmware.
I am having serious issues, w/ my household **wireless** devices. not wanting to pull downstream data.
Anything connected via Ethernet works perfect. It's just the wireless devices.
And no, I didn't change any settings/configurations after updating.

I have not had time to look into it very deeply myself, but I will as soon as I get the chance.
![image](https://github.com/gnuton/asuswrt-merlin.ng/assets/148441928/a1c536e2-3707-49b8-a60c-f3e59f7df0e8)
![image](https://github.com/gnuton/asuswrt-merlin.ng/assets/148441928/7cdbdeb8-5fb6-4b07-a59b-9f3f9543ae35)

**Comment by kyalejamison** [comment]
Not sure if this is a bug or just on my system, but i have 3  axe5q nodes and all my clients only will connect to the master even though the other 2 are connected and closer. And also in the ai mesh interface its not showing any clients listed for any node. 
![Screenshot (1)](https://github.com/gnuton/asuswrt-merlin.ng/assets/85897619/a047151c-e614-4f89-8d59-4ea849e48783)

![Screenshot (2)](https://github.com/gnuton/asuswrt-merlin.ng/assets/85897619/ada45a21-0bc9-4a23-92ff-6ec20ca9d9db)

**Reply by gnuton** [reply]
There is something not right here.
you do not have any client connected.
if you press F12 I guess you would see some errors in the javacscipt console

**Reply by kyalejamison** [reply]
you mean all these?
![Screenshot (3)](https://github.com/gnuton/asuswrt-merlin.ng/assets/85897619/cc2824d6-11c2-48ab-b5e9-5bd6cb3677f8)

**Comment by cseryz** [comment]
Hello all,
Thanks for your efforts.
Got 2 questions please.
Recently purchased a Ax58u v2, its runnig on latest formware (3.0.0.4.388_25127). If I install this port of merlin fw, will it keep.my settings ? Or I have to make a reset to defaults ?
A send one is, is that a plan to make new build based on latest asus firmware ?
Thanks

**Comment by gnuton** [comment]
Hi, the new firmware will be released in December. We depend on Asus for
the GPLs.  Please reset to factory after flashing the new firmware

**Comment by Ramonline** [comment]
Cool thanks ill test it on a   TUF-AX3000 V2
actually searching for it thanks
